dev/AppNotifications/AppNotificationProgressData.cpp

```diff
@@ -4,6 +4,12 @@
 
 namespace winrt::Microsoft::Windows::AppNotifications::implementation
 {
+    AppNotificationProgressData::AppNotificationProgressData(uint32_t sequenceNumber)
+    {
+        auto lock{ m_lock.lock_exclusive() };
+        THROW_HR_IF(E_INVALIDARG, sequenceNumber == 0); // The sequence number is always greater than 0
```

Comment:
This could be moved before the lock

Reply:
Actually there is no point in having a lock in consructors. So removing it there. However, I will fix this in the getters/setters